Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.cpp

```diff
@@ -440,14 +442,14 @@ void ResourceLoadStatisticsStore::removeDataRecords(CompletionHandler<void()>&&
     });
 }
 
-void ResourceLoadStatisticsStore::processStatisticsAndDataRecords()
+void ResourceLoadStatisticsStore::processStatisticsAndDataRecords(CompletionHandler<void()>&& completionHandler)
```

Comment:
Do you need to add this completion handler in this change? It's not clear to me why this is related to extending the days on script-written data delation.